config/initializers/geokit_config.rb

```diff
@@ -0,0 +1,100 @@
+# These defaults are used in Geokit::Mappable.distance_to and acts_as_mappable
+Geokit::default_units = :miles # others :kms, :nms, :meters
```

Comment:
@faissalMT - is the Redis cache enabled for this as was for the `geocoder` gem?

Reply:
Just spoken to @misaka about this and perhaps not so important to have the caching enabled now that the geocoding backfilling has been done